zenodo/modules/records/data/objecttypes.json

```
@@ -194,6 +194,25 @@
   },
   "csl": "graphic"
 },
+{
+  "$schema": "https://zenodo.org/schemas/records/objecttype-v1.0.0.json",
+  "internal_id": "workflow",
+  "id": "https://zenodo.org/objecttypes/workflow#",
+  "title": {
+    "en": "Workflow"
+  },
+  "title_plural": {
+    "en": "Workflows"
+  },
+  "datacite": {"general": "Workflow"},
+  "eurepo": "info:eu-repo/semantics/other",
+  "schema.org": "https://schema.org/CreativeWork",
+  "openaire": {
+    "resourceType": "0020",
+    "type": "other"
+  },
+"csl": "article"
```

```suggestion
"csl": "software"
```

More appropriate, but not sure if it will make a difference in citation rednering.